utils/handler.go

```diff
@@ -4,15 +4,20 @@ import (
 	"io"
 	"net"
 	"net/http"
+
+	log "github.com/sirupsen/logrus"
 )
 
 type ErrorHandler interface {
 	ServeHTTP(w http.ResponseWriter, req *http.Request, err error)
 }
 
-var DefaultHandler ErrorHandler = &StdHandler{}
+var DefaultHandler ErrorHandler = &StdHandler{
+	log: log.StandardLogger(),
+}
```

Comment:
Is this logger used anywhere?  I couldn't find any usages.  (Except in this file, but no calls to any `log.*` function.)